fedimint-api-client/src/api/mod.rs

```diff
@@ -855,20 +895,36 @@ impl ConnectorRegistry {
             None => url,
         };
 
-        let Some(connector) = self.connectors.get(url.scheme()) else {
+        let connector_key = match url.scheme() {
+            // ws and wss are handled by the same connector instance
+            "ws" | "wss" => "ws",
```

Comment:
Eventually the key should probably be a list of schemas that we can match against to avoid such hardcoded hacks